apps/web-twig-demo/assets/scripts/toast-dynamic.ts

```diff
@@ -6,11 +6,16 @@ const addDynamicToast = (event, containerId) => {
     autoCloseInterval: formElement.querySelector('#toast-auto-close-interval').value,
     color: formElement.querySelector('#toast-color').value,
     containerId,
-    content: formElement.querySelector('#toast-content').value,
     enableAutoClose: formElement.querySelector('#toast-enable-auto-close').checked,
+    enableLink: formElement.querySelector('#toast-enable-link').checked,
```

Comment:
do we actually need to pass this here? (and in web package too)?

Reply:
IMHO, yes, because Toast.ts determines whether the element will be updated (attributes will be added) or removed from the template (DOM)

Reply:
updated